src/components/Section/Section.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Box, { BoxProps } from '../Box';
+import { HeadingLevelContext } from '../H';
+
+/**
+ * Increases the heading level of all `H` components inside it by 1.
+ */
+export const Section: React.FC<BoxProps<'section'>> = ({ children, as = 'section', ...rest }) => {
+  const level = React.useContext(HeadingLevelContext);
```

Comment:
q: I see that we have a default context value, but I'm a bit baffled by something. Can't `level` never be `undefined` if there's no parent provider? 

I see that the related tests pass, but I'm failing to understaand why. In my mind useContext returns `undefined` if  no provider parent exists, but I see that somehow in your tests that's not the case

Reply:
> Can't level never be undefined if there's no parent provider?

Nope! The default value to `createContext(1)` takes care of that!

> In my mind useContext returns undefined if no provider parent exists

This is slightly off from the way `Context.Consumer` & `useContext` works. If the context is given a default value, that will be provided by `useContext` in the event that it's rendered without a provider.

[React docs:](https://reactjs.org/docs/context.html#reactcreatecontext)

> The defaultValue argument is only used when a component does not have a matching Provider above it in the tree.